test/contrib/bigquery_gcloud_test.py

```diff
@@ -31,14 +31,20 @@
 from contrib import gcs_test
 from nose.plugins.attrib import attr
 
+from testfixtures import should_raise
```

Comment:
We can avoid the new testfixtures dependency - nose has [`assertRaises`](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises)